TSE/CCExtensions.cpp

```diff
@@ -1656,18 +1656,34 @@ static PRIMITIVEPROCDEF g_Extensions[] =
 			"property (all)\n\n"
 
 			"   'ascended\n"
-			"   'category\n"
+			"   'category -> 'beam | 'effect | 'marker | 'missile | 'ship | 'station\n"
+			"   'commsKey\n"
+			"   'currency -> currency type UNID\n"
```

Comment:
Unfortunately, the spacing doesn't work on the forum (and you can't read it all in the dev console). I think this will be easier to read and clearly conveys, "this is what the function returns if you pass this argument." But I could be persuaded if you think it's really important to standardize.